src/components/trends/common/Tooltip.css

```diff
@@ -0,0 +1,40 @@
+.tooltip {
+  display: block;
+  font-size: 14px;
+  background-color: #FFFFFF;
+  border: solid;
+  position: absolute;
+  border-radius: 4px;
+  opacity: 0.75;
+}
+
+.content {
+  display: flex;
```

Comment:
There aren't yet, and there's no place that I could find that repeats more than a single flexbox style across our CSS yet, so there'd be no real benefit in making them composable classes (they'd have to be so granular right now that we'd just be repeating the CSS in a new way).